core/tests/protractor_desktop/blogDashboard.js

```diff
@@ -0,0 +1,192 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+// //
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// //
+//      http://www.apache.org/licenses/LICENSE-2.0
+// //
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the blog Dashboard page.
+ */
+
+var action = require('../protractor_utils/action.js');
+var forms = require('../protractor_utils/forms.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+var general = require('../protractor_utils/general.js');
+
+var BlogDashboardPage = require('../protractor_utils/BlogDashboardPage.js');
+
+describe('Blog dashboard functionality', function() {
+  var blogDashboardPage = null;
+
+  beforeAll(async function() {
+    blogDashboardPage = (
+      new BlogDashboardPage.BlogDashboardPage());
+    await users.createUserWithRole(
+      'blog@blogDashboard.com',
+      'blog',
+      'blog admin');
+    await users.login('blog@blogDashboard.com');
+  });
+
+  beforeEach(async function() {
+    await blogDashboardPage.get();
+  });
+
+  it('should check that editor name and profile photo is visible',
```

Comment:
The string here should describe what the blog dashboard page should do, not what the test should do

ditto for the other test names

Reply:
```suggestion
  it('should check that editor name and profile photo are visible',
```

Reply:
Done.